app/controllers/seances_controller.rb

```diff
@@ -0,0 +1,56 @@
+class SeancesController < ApplicationController
+  def index
+    seances = Seance.includes(:movie).filtered_by_day(date).group_by(&:movie_title)
+    # seances = Seance.includes(:movie).group_by(&:movie_title)
+
+    render :index, locals: { seances: }
+  end
+
+  def show
+    render :show, locals: { seance: }
+  end
+
+  def new
+    render :new, locals: { seance: Seance.new }
+  end
+
+  def edit
+    render :edit, locals: { seance: }
+  end
+
+  def create
+    record = Seance.new(seance_params)
+    if record.save
+      redirect_to record, notice: t('.notice')
+    else
+      render :new, locals: { seance: record }
+    end
+  end
+
+  def update
+    if seance.update(seance_params)
+      redirect_to seance, notice: t('.notice')
+    else
+      render :edit
```

Comment:
I think you miss locals here. Please check also `movies#update` action.

Reply:
you're right, I should pass locals here as well